### PR TITLE
Add dracula style for tab-bar and tab-line faces

### PR DESCRIPTION
**Pull request by tsdh**
One new Emacs 27 feature is a native tab-bar and window tab-lines.  This patch
gives them a dracula style.

### CODE REVIEW

**Comment by tsdh** [comment]
Here's the mandatory screenshot.

Screenshot-20191017130451-778x518

**Review by milouse — CHANGES_REQUESTED** [review]
Hi, and thank you for your two merge requests!

On this one, I have some comments before merging it. Can you fix the indentation of the lines you added by using only spaces (not tabulations)?

And to be sure, I'm not quite sure of how each new face is illustrated in your screenshot, can you annotate your screenshot to show what is the tab-bar, the tab-line, an inactive tab-line-tab and an inactive tab-bar-tab?

Thanks again!

**Comment by tsdh** [comment]
@milouse I've replaced tabs with spaces and added a file local variable turning off indent-tabs-mode, too.  Also, I had to redo most of my changes as many color variables were renamed.  One color has been renamed and then dropped with the commit where powerline faces were removed.  I've added it again.

I cannot edit the screenshot but I can explain in words.  The tab-bar is the bar with tabs on top of the emacs frame.  The tab-lines are the tabs on top of each emacs window.  As you can see, the tab names in the tab-lines are a bit smaller than in the tab-bar.

The tab title face (tab-bar-tab and tab-line-tab) is lighter for the currently active tab (sort of magenta) and darker (sort of violet) for the inactive tabs (tab-bar-tab-inactive, tab-line-tab-inactive).

**Comment by milouse** [comment]
Ok understood. The behavior seems a bit strange (with the two tab bar on the top of the window, but it's hard to have an opinion on that without test I think).

Thank you for the change regarding space and the local variable.

Yes, sorry, I've continue to work on the theme to make it more easy to maintain, leading to a huge deletion of color names, which were duplicated.

I'm not sure you should use the remove `other-violet` color, which is not in the upstream color panel. Is there a reason you don't want to take the `dracula-purple` ?

Otherwise, do you know if the (x) and [+] button to close or add tabs) are themeable?

Anyway, it's a good start for this new feature, thus I may merge it as-this and wait for general delivery to improve it if other have other ideas. I'm just waiting for your answer for my 2 above question in case you can do something.

Again, thank you very much for your help!

**Comment by milouse** [comment]
I finally take time to compile emacs 27.

As shown in the capture bellow, I think `dracula-purple` is ok, thus I'll merge with that to avoid to introduce a new color.

I found no easy way to customize the button, thus I leave it as this for now too.

Capture du 2019-10-28 19-59-22

**Review by milouse — APPROVED** [review]

**Comment by milouse** [comment]
Closed by 9ca1ad9

**Comment by tsdh** [comment]
@milouse dracula-purple is fine, too.  Thanks for merging!

**Pull request closed without merging**
